cmd/oras/internal/display/status/utils.go

```diff
@@ -52,12 +52,6 @@ const (
 const (
 	IndexPromptFetching = "Fetching "
```

Comment:
Extra space, same for fetched.

```suggestion
	IndexPromptFetching = "Fetching"
```

Reply:
We are not doing status output indentation align anymore?